### PR TITLE
Adding support for comment in table/view

### PR DESCRIPTION
**Pull request by sharadgaur**
Adding Table/View Comment Support as per document https://clickhouse.com/docs/en/sql-reference/statements/create/table
https://github.com/AfterShip/clickhouse-sql-parser/issues/105
Example:
```
CREATE TABLE IF NOT EXISTS test.events_local (
    f0 String,
    f1 String CODEC(ZSTD(1)),
    f2 VARCHAR(255),
    f3 Datetime,
    f4 Datetime,
    f5 Map(String,String),
    f6 String,
    f8 Datetime DEFAULT now(),
    f9 String MATERIALIZED toString(f7['f70']),
    f10 String ALIAS f11,
) ENGINE = MergeTree
PRIMARY KEY (f0, f1, f2)
PARTITION BY toYYYYMMDD(f3)
TTL f3 + INTERVAL 6 MONTH
ORDER BY (f1,f2,f3)
COMMENT 'Comment for table';

### CODE REVIEW

File: parser/parser_test.go [hunk 1]
```diff
@@ -72,7 +72,8 @@ func TestParser_ParseStatements(t *testing.T) {
 }
 
 func TestParser_Format(t *testing.T) {
-	for _, dir := range []string{"./testdata/dml", "./testdata/ddl", "./testdata/query", "./testdata/basic"} {
+	for _, dir := range []string{"./testdata/ddl"} {
```
Comment (git-hulk) [inline parser/parser_test.go:75]: @sharadgaur We should revert this change?

**Comment by git-hulk** [comment]
Generally looks good, just one comment inline. Thank you!

File: parser/parser_test.go (hunk 1 above)
Comment (sharadgaur) [reply parser/parser_test.go:75]: ah let me fix it

File: parser/parser_test.go [hunk 2]
```diff
@@ -83,6 +83,9 @@ func TestParser_Format(t *testing.T) {
 			if !strings.HasSuffix(entry.Name(), ".sql") {
 				continue
 			}
+			if entry.Name() != "create_materialized_view_basic.sql" {
+				continue
+			}
```
Comment (git-hulk) [inline parser/parser_test.go:88]:
```suggestion
```

**Review by git-hulk — APPROVED** [review]
